### PR TITLE
add podcast page styling

### PR DESCRIPTION
**Pull request by rccsousa**
Why:
- podcast pages were missing styling;
How:
- added styling to the different blocks of the podcast page except for the `AudioPlayer` which is comming with a custom style on a future PR;

### CODE REVIEW

**Comment by rccsousa** [comment]
@davelange must update mobile layouts 🤦 converting back to draft

**Comment by rccsousa** [comment]
done ✅

File: src/app/_blocks/EpisodeHead/AudioPlayer/Buttons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+export function PlayButton() {
+  return (
+    <svg width="54" height="54" viewBox="0 0 54 54" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <circle cx="27" cy="27" r="27" fill="white" />
+    </svg>
+  )
+}
+
+;<svg
```
Comment (davelange) [inline src/app/_blocks/EpisodeHead/AudioPlayer/Buttons/index.tsx:9]: Is this svg lost?

**Review by davelange — APPROVED** [review]

File: src/app/_blocks/EpisodeHead/AudioPlayer/Buttons/index.tsx (hunk 1 above)
Comment (rccsousa) [reply src/app/_blocks/EpisodeHead/AudioPlayer/Buttons/index.tsx:9]: No, it was the preparation for the upcoming branch / custom audio player (but got caught up before branching out 😅 )

**Pull request merged**